### DISCUSSION TITLE
Is there any method to open links in mkdocs material in a new tab?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#6025] [Q&A] [answered]
Is there any method to open links in mkdocs material in a new tab? Now, when l open a link inside the mkdocs material, it will be opened in this original tab, which is not convenient for me because if l want to go back to the mkdocs but not close the tab l have opened. So, l am wondering if there is any method to open a link in a new tab in mkdocs material?

Thanks for helping!

**Comment by lampensau** [comment, marked as answer]
Yes, I believe there are two ways you could go about this:

1. Use the [attr_list](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/?h=attr#attribute-lists) extension and add something like `{target="_blank"}` to your links (e.g., `[link text](https://target.link){target="_blank"}`).
2. Use the privacy plugin's [external link](https://squidfunk.github.io/mkdocs-material/plugins/privacy/?h=blank#external-links) functionality (requires insiders)

**Reply by mendax1234** [reply]
Thanks!

**Reply by kamilkrzyskow** [reply]
The Privacy plugin is a part of the Insiders version, and will be in the community version after the next funding goal.

For the time being you can also check out other plugins / hooks that do this job here  
https://github.com/squidfunk/mkdocs-material/discussions/3660

Apart of the pre-build / on-build methods, you can also add the attribute post-build via some extra JavaScript.

**Comment by z-aki** [comment]
wrote a plugin to simplify this https://github.com/z-aki/mkdocs-plugin-open-external-links-in-new-tab/
